zwave-classifier.js

```diff
@@ -23,6 +23,22 @@ const COMMAND_CLASS_METER = 50;               // 0x32
 //const COMMAND_CLASS_SWITCH_ALL = 39;        // 0x27
 const COMMAND_CLASS_CONFIGURATION = 112;    // 0x98
 
+const QUIRKS = [
+  {
+    identifiers: {
```

Comment:
Since we're matching this against zwInfo from the node, I think it would be clearer if this was also called zwInfo